common/presenters/population-table/population-table-config.js

```diff
@@ -0,0 +1,73 @@
+const { addDays, format, isSameDay } = require('date-fns')
+
+const locationMetaToSpacesLink = require('./location-meta-to-spaces-link')
+
+const populationTableDay = {
+  establishmentConfig({ isSortable = false }) {
+    return {
+      head: {
+        isSortable,
+        sortKey: 'title',
+        html: 'Establishment',
+        attributes: {
+          width: '220',
+        },
+      },
+      row: {
+        attributes: {
+          scope: 'row',
+        },
+        html: data => {
+          return `<a>${data.title}</a>`
```

Comment:
I'm guessing the link is just a placeholder at this stage?

Reply:
Yeah, as I said in the description:

> The links to establishments and spaces are currently not useful - these will be improved by the subsequent ticket involving these pieces of functionality.

Until I put the routing in, the links aren't useful. So that the PR is simpler, I've kept the `<a>` tag, but I've removed all the properties